apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -218,6 +186,11 @@ private void stop() {
 
     }
 
+    private void setWorkloadHigh(int workloadHigh) {
+        mWorkloadHigh = workloadHigh;
+        mWorkloadLow = Math.max(1, (int)(workloadHigh * 0.016));
```

Comment:
Any reason why we have both .016 and LOAD_RECOVERY_HIGH?

Reply:
Now I just set it to 1.